tests/validation/tests/v3_api/test_eks_v2_hosted_cluster.py

```diff
@@ -114,6 +116,7 @@ def test_eks_v2_hosted_cluster_create_all():
     validate_nodegroup(eks_config_temp["nodeGroups"], cluster_name)
 
 
+@pytest.mark.skip(reason="skipping for now, only run basic cluster configuration test")
```

Comment:
Instead of just skipping could we use a bool to skip, or in the Jenkins job just call the `test_eks_v2_hosted_cluster_create_basic` test? That way test can be run if we ever need to.

Reply:
Yeah, I can remove the skips. I think we actually are calling just that test, so this isn't needed anymore.

Reply:
Resolved

Reply:
sweet